### PR TITLE
Doc: add missing sections.

### PR DESCRIPTION
**Pull request by jmoreira-valory**
Doc: add missing sections.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -1,13 +1,78 @@
...
+	autonomy fetch valory/elcollectooorr:0.1.0:bafybeidzyd2yi3igjinpi7bwnyal235bmvovuyxy4qrxnitah6fqysemxq --service
+	```
+
+2. Build the Docker image of the service agents
+
+	```bash
+	cd elcollectooorr
+	autonomy build-image
+	```
+
```
Comment (0xArdi) [inline docs/index.md:25]: Maybe it makes sense to add the env variables here?

Comment (jmoreira-valory) [reply docs/index.md:25]: Please suggest change.

File: docs/index.md [hunk 2]
```diff
@@ -1,13 +1,78 @@
...
 
-##Build your own
+1. Fetch the El Collectooorr service.
 
-!!! info
-	This section will be added soon.
+	```bash
+	autonomy fetch valory/elcollectooorr:0.1.0:bafybeidzyd2yi3igjinpi7bwnyal235bmvovuyxy4qrxnitah6fqysemxq --service
+	```
+
```
Comment (0xArdi) [inline docs/index.md:18]:
```suggestion
      Ensure you have set the following environment variables:

      ```bash
      export SAFE_CONTRACT_ADDRESS=`["0x123a3d66cf688b676f9b7a6bcc3991f62fec7f0a"]`
      export WHITELISTED_INVESTOR_ADDRESSES='["YOUR_WHITELIST"]'
      export SERVICE_ELCOLLECTOOORR_RPC_0="YOUR_RPC_URL"
      export SERVICE_ELCOLLECTOOORR_RPC_1="YOUR_RPC_URL"
      export SERVICE_ELCOLLECTOOORR_RPC_2="YOUR_RPC_URL"
      export SERVICE_ELCOLLECTOOORR_RPC_3="YOUR_RPC_URL"
      ```

      where `0x123a3d66cf688b676f9b7a6bcc3991f62fec7f0a` should match the correct address from the on-chain service deployment, and `YOUR_WHITELIST`, `YOUR_RPC_URL_0` and `YOUR_RPC_URL_1` should be replaced accordingly.

Note that `0x123a3d66cf688b676f9b7a6bcc3991f62fec7f0a` is the mainnet safe address of the El Collectooorr. You will be able to run the service by setting this address. However, any on-chain transaction will fail unless you have the operator keys.
```

**Review by 0xArdi — APPROVED** [review]

File: docs/index.md [hunk 3]
```diff
@@ -1,13 +1,94 @@
...
 
+Once you have {{set_up_system}} to work with the Open Autonomy framework, you can run a local demo of the El Collectooorr service as follows:
 
-##Build your own
+1. Fetch the El Collectooorr service.
 
-!!! info
-	This section will be added soon.
+	```bash
+	autonomy fetch valory/elcollectooorr:0.1.0:bafybeidzyd2yi3igjinpi7bwnyal235bmvovuyxy4qrxnitah6fqysemxq --service
```
Comment (DavidMinarsch) [inline docs/index.md:15]: This is not congruent with what's currently in `packages.json`

File: docs/index.md [hunk 4]
```diff
@@ -1,13 +1,94 @@
...
+The MintKit is a toolkit to build services with minting capabilities. For example, El Collectooorr service, which is based on the MintKit, aims to autonomously create collections of NFTs by minting them as they drop on Art Blocks. It uses complex logic to decide which mints to participate in and what prices to bid at, so users don’t have to. The service is fueled by the community of users who fund collections. Each collection is designed to have its own unique ERC-20 token, for users to vote and collectively decide on what happens to the collection.
 
+## Demo
 
-##Run the code
+!!! warning "Important"
 
-!!! info
-	This section will be added soon.
+	This section is under active development - please report issues in the [Autonolas Discord](https://discord.com/invite/z2PT65jKqQ).
```
Comment (DavidMinarsch) [inline docs/index.md:8]: Please make sure the link is non-expiring

Comment (jmoreira-valory) [reply docs/index.md:8]: This is the usual discord link, which I believe is not expiring. In any case, I'll add open a task for the Server Owner on the related document to double-check it.

File: docs/index.md (hunk 3 above)
Comment (jmoreira-valory) [reply docs/index.md]: Fixed.

**Pull request merged**